src/TodoContext.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable max-len */
+
+import { createContext } from 'react';
+import { TodoContextProps } from './types/interfaces';
+
+export const TodoContext = createContext<TodoContextProps | undefined>(undefined);
```

Comment:
interesting solution, but is that good, that you have so big `App` component because of it?

Reply:
its the first task where tried to use 'one' Context. I was lazy to write again useReducer, like in prev Todo App with localStorage.

Reply:
and all in App - just simplified task for myself

Reply:
Ah, I understood, need to see your finished todoapp)